homework/main.ipynb

```diff
@@ -2,45 +2,45 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "--2024-10-16 10:31:30--  https://data.insideairbnb.com/spain/catalonia/barcelona/2024-09-06/data/listings.csv.gz\n",
+      "--2024-10-21 17:05:04--  https://data.insideairbnb.com/spain/catalonia/barcelona/2024-09-06/data/listings.csv.gz\n",
       "Resolving data.insideairbnb.com (data.insideairbnb.com)... 108.138.51.28, 108.138.51.112, 108.138.51.6, ...\n",
       "Connecting to data.insideairbnb.com (data.insideairbnb.com)|108.138.51.28|:443... connected.\n",
       "HTTP request sent, awaiting response... 200 OK\n",
       "Length: 9977937 (9.5M) [application/x-gzip]\n",
       "Saving to: ‘listings.csv.gz’\n",
       "\n",
-      "listings.csv.gz     100%[===================>]   9.52M  25.8MB/s    in 0.4s    \n",
+      "listings.csv.gz     100%[===================>]   9.52M  27.0MB/s    in 0.4s    \n",
       "\n",
-      "2024-10-16 10:31:31 (25.8 MB/s) - ‘listings.csv.gz’ saved [9977937/9977937]\n",
+      "2024-10-21 17:05:04 (27.0 MB/s) - ‘listings.csv.gz’ saved [9977937/9977937]\n",
       "\n",
-      "--2024-10-16 10:31:31--  https://data.insideairbnb.com/spain/catalonia/barcelona/2024-09-06/data/calendar.csv.gz\n",
-      "Resolving data.insideairbnb.com (data.insideairbnb.com)... 108.138.51.28, 108.138.51.112, 108.138.51.6, ...\n",
-      "Connecting to data.insideairbnb.com (data.insideairbnb.com)|108.138.51.28|:443... connected.\n",
+      "--2024-10-21 17:05:05--  https://data.insideairbnb.com/spain/catalonia/barcelona/2024-09-06/data/calendar.csv.gz\n",
+      "Resolving data.insideairbnb.com (data.insideairbnb.com)... 108.138.51.112, 108.138.51.6, 108.138.51.93, ...\n",
+      "Connecting to data.insideairbnb.com (data.insideairbnb.com)|108.138.51.112|:443... connected.\n",
       "HTTP request sent, awaiting response... 200 OK\n",
       "Length: 18042988 (17M) [application/x-gzip]\n",
       "Saving to: ‘calendar.csv.gz’\n",
       "\n",
-      "calendar.csv.gz     100%[===================>]  17.21M  27.2MB/s    in 0.6s    \n",
+      "calendar.csv.gz     100%[===================>]  17.21M  30.1MB/s    in 0.6s    \n",
```

Comment:
Do we really need any file except `reviews.csv.gz`?

Reply:
We need listings, and probably reviews. The calendar is just in case)